content/enterprise_influxdb/v1.9/install-and-deploy/installation/meta_node_installation.md

```diff
@@ -172,33 +186,32 @@ On systemd systems, enter:
 sudo systemctl start influxdb-meta
 ```
 
-> **Verification steps:**
->
+#### Step 2b: Verification steps
```

Comment:
think we can drop Step 2b here and say "Verify....

Reply:
Good catch! 👍